bci_tester/data.py

```diff
@@ -365,6 +365,7 @@ def create_BCI(
     for ver, tag in (("15.6", "9.24"), ("tumbleweed", "10.0"))
 ]
 
+BASE_FIPS_CONTAINERS = []
```

Comment:
Does it make things simpler if we switched this to a single constant?

Reply:
you mean not a list but a single parameterset? it was that originally, but there are OS_VERSIONS for which none exists so it's easier to handle the "test this container if it exists otherwise do nothing" approach by having a list with a single entry or an empty list. less code duplication imho.